frontend-react/src/pages/submissions/SubmissionsTable.tsx

```diff
@@ -1,17 +1,78 @@
-import { useContext } from "react";
+import { useContext, useState } from "react";
 import moment from "moment";
 import { useResource } from "rest-hooks";
+import {
+    Button,
+    IconNavigateBefore,
+    IconNavigateNext,
+} from "@trussworks/react-uswds";
 
 import SubmissionsResource from "../../resources/SubmissionsResource";
 import { GlobalContext } from "../../components/GlobalContextProvider";
 
 function SubmissionsTable() {
     // this component will refresh when global context changes (e.g. organization changes)
     const globalState = useContext(GlobalContext);
+
+    // state of pagination
+    const [paginationCursor, setPaginationCursor] = useState("");
+    const [paginationSort, setPaginationSort] = useState("DESC");
+    const [paginationPageSize] = useState(10);
+
     const submissions: SubmissionsResource[] = useResource(
         SubmissionsResource.list(),
-        { organization: globalState.state.organization }
+        {
+            organization: globalState.state.organization,
+            cursor: paginationCursor,
+            pageSize: paginationPageSize,
+            sort: paginationSort,
+        }
     );
+
+    const sortedSubmissions = () => {
+        if (paginationSort === "ASC") {
+            // sort by createdAt DESC
+            // because when the paginationSort is ASC, the results from the server are reversed
+            submissions.sort(
+                (a, b) =>
+                    Date.parse(b.createdAt?.toString() || "") -
+                    Date.parse(a.createdAt?.toString() || "")
+            );
+        }
+        return submissions;
+    };
+
+    const updatePaginationCursor = (next: boolean) => {
+        let cursor = submissions[0]?.createdAt?.toString() || "";
+        let sort = "ASC";
+        if (next) {
+            cursor =
+                submissions[submissions.length - 1]?.createdAt?.toString() ||
```

Comment:
Could `submissions.length` be `0`... in which case you're accessing `submissions[-1]`